pkg/api/v1beta3/dynakube/validation/activegate.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
we want the validation logic to be versioned, so pls not move it, but copy it

we can consider moving the common things (this will "never ever change") in a common area, but not right now.

Reply:
okey, will do.

Reply:
fixed in 38d79f13166e27799292383d0c9c3571f6a58be5